eventmesh/ApiServerEventMesh.js

```diff
@@ -0,0 +1,303 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const assert = require('assert');
+const config = require('../common/config');
+const logger = require('../common/logger');
+const CONST = require('../common/constants');
+const EventMeshServer = require('./EventMeshServer');
+const kc = require('kubernetes-client');
+const JSONStream = require('json-stream');
+
+
+const apiserver = new kc.Client({
+  config: {
+    url: `https://${config.internal.ip}:9443`,
+    insecureSkipTlsVerify: true
+  },
+  version: '1.9'
+});
+
+class ApiServerEventMesh extends EventMeshServer {
+  registerWatcher(resourceName, resourceType, callback) {
+    return Promise.try(() => apiserver.loadSpec())
+      .then(() => {
+        const stream = apiserver
+          .apis[`${resourceName}.servicefabrik.io`]
```

Comment:
`.servicefabrik.io` should also be a constant as the apiserver host name.

Reply:
Done!